tests/kernel/install_ltp.pm

```diff
@@ -275,6 +276,10 @@ sub setup_network {
     # echo/echoes, getaddrinfo_01
     assert_script_run('sed -i \'s/^\(hosts:\s+files\s\+dns$\)/\1 myhostname/\' /etc/nsswitch.conf');
 
+    my $firewall = 'firewalld';
```

Comment:
No need to specify firewalld or SuSEfirewall2. You have that set in `opensusebasetest::firewall` since you are using 
`use base 'opensusebasetest';`. 

So, I recommend doing something like:

    disable_and_stop_service(opensusebasetest::firewall);

Anyway, it might cause some problems in QAM jobs. We faced the same situation in Wicked. See
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/wicked/before_test.pm#L37
@mdoucha

Reply:
@mdoucha in the end removed your commit entirely (use simple `disable_and_stop_service(opensusebasetest::firewall);` as @jlausuch suggested).

@jlausuch IMHO wicked on QAM is ok: whole bug is WONTFIX on QAM, thus QAM marked this bug as a soft failure with their yaml https://openqa.suse.de/tests/3568323#step/dhcpd6/8

Reply:
@pevik The only bug that's soft-failed is the wrong IPv6 prefix. Any other failure (test program exit code) will hard fail. https://openqa.suse.de/tests/3586536

Reply:
@mdoucha Thanks for info. I'll keep the code and wrap it temporarily with `if is_released`.